src/conf_mode/service_ssh.py

```diff
@@ -95,6 +95,16 @@ def generate(ssh):
         syslog(LOG_INFO, 'SSH ed25519 host key not found, generating new key!')
         call(f'ssh-keygen -q -N "" -t ed25519 -f {key_ed25519}')
 
+    if 'trusted_user_ca_key' in ssh:
+        if ssh['trusted_user_ca_key'].startswith('http'):
+            result = call(f"curl --fail --silent -o /etc/ssh/trusted_user_ca_key {ssh['trusted_user_ca_key']}")
+            if result != 0:
+                raise RuntimeError("Failed to download TrustedUserCAKeys file")
+        else:
+            if not os.path.isfile(ssh['trusted_user_ca_key']):
```

Comment:
This check should be done in `verify()` step

Reply:
[eed5816](https://github.com/vyos/vyos-1x/pull/4234/commits/eed5816277e3c7cc59d7c5d67f267ab78d0586d7)
I've put together a fix for it here:) 🙏